test/engagement-changeset-aware.e2e-spec.ts

```diff
@@ -120,11 +121,18 @@ const readProjectChangeset = (app: TestApp, id: string, changeset?: string) =>
   );
 
 const activeProject = async (app: TestApp) => {
-  const fundingAccount = await createFundingAccount(app);
-  const location = await createLocation(app, {
-    fundingAccountId: fundingAccount.id,
+  const fundingAccount = await runInIsolatedSession(app, async () => {
+    await registerUser(app, { roles: [Role.Administrator] }); // only admin can create funding account for now
+    return await createFundingAccount(app);
+  });
+  const location = await runInIsolatedSession(app, async () => {
+    await registerUser(app, { roles: [Role.Administrator] }); // only admin can create funding account for now
+    return await createLocation(app, { fundingAccountId: fundingAccount.id });
+  });
+  const fieldRegion = await runInIsolatedSession(app, async () => {
+    await registerUser(app, { roles: [Role.Administrator] }); // only admin can create funding account for now
+    return await createRegion(app);
```

Comment:
Seeing a lot of these. You can use `runAsAdmin` instead of `runInIsolatedSession`. It will login as the root admin instead of registering a new user every time.

Also merge these together when possible, so we don't waste session & login queries.

```suggestion
  // only admins can create these
  const [fundingAccount, location, region] = await runAsAdmin(app, async () => {
    const fundingAccount = await createFundingAccount(app);
    const location = await createLocation(app, { fundingAccountId: fundingAccount.id });
    const region = await createRegion(app);
    return [fundingAccount, location, region];
```

Reply:
This suggested change also doesn't work